modules/report/view/reskontrovoucherprint.php

```diff
@@ -381,7 +380,7 @@ function get_saldo($account, $fromperiod, $toperiod)
                     }
                     ?></td>
                 <td class="noprint align-right"><nobr><?
-                  $amount_to_show = $postmotpost->getDiff($voucher->AccountPlanID, $voucher->KID, $voucher->InvoiceID, $match_number, $voucher);
+                  $amount_to_show = $reconciliation->getDiff($voucher->AccountPlanID, $voucher->KID, $voucher->InvoiceID, $match_number, $voucher);
```

Comment:
`$match_number = $_lib['db']->get_row(array("query" => "SELECT MatchNumber FROM vouchermatch WHERE VoucherID = $voucher->VoucherID"))->MatchNumber;`
change to 
`$match_number = $voucher->MatchNumber;`